src/js/components/PlacementConstraintsPartial.js

```diff
@@ -211,7 +216,7 @@ export default class PlacementConstraintsPartial extends Component {
 
     return (
       <div>
-        {this.getPlacementConstraintsFields(data.constraints)}
+        {this.getPlacementConstraintsFields(data.constraints, i18n)}
```

Comment:
Alternatively you could pull i18n from props inside getPlacementConstraintsFields